test/conformance/kuttl/verifyImages/clusterpolicy/standard/update-multi-containers/policy.yaml

```diff
@@ -0,0 +1,31 @@
+apiVersion: kyverno.io/v1
+kind: ClusterPolicy
+metadata:
+  name: verify-image-signature
+spec:
+  background: false
+  failurePolicy: Fail
+  rules:
+  - match:
+      any:
+      - resources:
+          kinds:
+          - Pod
+    name: verify-image
+    verifyImages:
+    - attestors:
+      - count: 1
+        entries:
+        - keys:
+            publicKeys: |
+              -----BEGIN PUBLIC KEY-----
+              MFkwEwYHKoZIzj0CAQYIKoZIzj0DAQcDQgAEMKLYTatU9CUsrA5Td6jXiZTolwsx
+              HZKwYP5XkHhU436FGDD5Zi2nVFem6AbzXWHssIQRkAI3yJgKkB4J6Qe4OQ==
+              -----END PUBLIC KEY-----
+      imageReferences:
+        - ghcr.io/seankhliao/*
```

Comment:
can the test use images owned kyverno?
I'm not promising that I won't forget and accidentally remove these in a cleanup at some point